foundation/foundation-transform/aissemble-foundation-transform-core-python/pom.xml

```diff
@@ -20,6 +20,9 @@
             <plugin>
                 <groupId>org.technologybrewery.habushu</groupId>
                 <artifactId>habushu-maven-plugin</artifactId>
+                <configuration>
+                    <rewriteLocalPathDepsInArchives>true</rewriteLocalPathDepsInArchives>
+                </configuration>
```

Comment:
I: This follow's the resolution from model-training-api for installing packages within mono repos.